array-methods/every.js

```diff
@@ -0,0 +1,23 @@
+//const Age  = (majeur) => majeur >= 18;
+//const arrayAge= [18, 20, 19, 30, 45];
+//console.log(arrayAge.every(Age));
+
+//Exemple d'une fonction qui représente le fonctionnement de array.every
+//Retourne vrai si l'évaluation de chacune des valeurs du tebleau répond au critère d'évaluation
+
+function MajeurEtVaccine(arrOfAge, majeur)
+{                  
+    let unAdulte = true;
```

Comment:
Ici tu aurais pu simplement faire:
`
    for (let i = 0; i < arrOfAge.length; i++) {        
        if (arrOfAge[i] < majeur) return false;
    }    
    return true;
`